### DISCUSSION TITLE
how can i migrate data from milvus lite to milvus standalone

### DISCUSSION THREAD

**Discussion by zlpmetyou** [milvus-io/milvus#36059] [Q&A and General discussion] [answered]
how can i migrate data from milvus lite to milvus standalone

**Comment by yhmo** [comment]
milvus-lite is a bit special, not able to use the milvus-backup tool to do backup/restore. You can use [queryIterator](https://milvus.io/docs/with-iterators.md#Query-with-an-iterator) to fetch data from milvus-lite and insert into milvus standalone.

**Comment by xiaofan-luan** [comment]
@liliu-z 
we should add a dump command to dump milvus list data and import to a milvus cluster

**Reply by liliu-z** [reply]
Nice suggestion, will take a closer look

**Comment by junjiejiangjjj** [comment, marked as answer]
hi @zlpmetyou , milvus-lite provides a tool to dump data, which can then be imported into Milvus via bulkinsert, https://github.com/milvus-io/milvus-lite?tab=readme-ov-file#migrating-data-from-milvus-lite

**Reply by liliu-z** [reply]
Should we include bulkinsert inside the cmd

**Reply by zlpmetyou** [reply]
i will try it later

**Reply by zlpmetyou** [reply]
Thank you for your support!
